lib/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.d.ts
```

Comment:
why are .d.ts files ending up here? they should not get mixed in with sources - your compiler output ought to go to a .gitignore'd folder like built/